uninstall.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Plugin uninstall handler.
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+use Google\Web_Stories\Story_Post_Type;
+use Google\Web_Stories\Template_Post_Type;
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+if ( ! defined( 'WP_UNINSTALL_PLUGIN' ) ) {
+	return;
+}
+
+/**
+ * Filters whether data should be erased when uninstalling the plugin.
+ *
+ * @param bool $erase Whether to erase data. Default false.
+ */
+$erase = (bool) apply_filters( 'web_stories_erase_data_on_uninstall', false );
+
+if ( false === $erase ) {
+	return;
+}
+
+require_once WEBSTORIES_PLUGIN_DIR_PATH . '/includes/uninstall.php';
+
+\Google\Web_Stories\delete_options();
+
+if ( is_multisite() ) {
+	\Google\Web_Stories\delete_site_options();
```

Comment:
On multisite, don't we need to delete posts, post meta and options for all sites on the multisite?

Reply:
Not sure, not a multisite expert 😕 

But looping through all sites would be super expensive. I don't think we wanna go down that road.

First thought:

Leave this as-is. BUT:

Once we have a settings page, allow deleting data via click of a button on that settings page. Like Site Kit does:

<img width="288" alt="Screenshot 2020-06-23 at 12 48 18" src="https://user-images.githubusercontent.com/841956/85395038-d5121f80-b54f-11ea-8a86-37184247e5d9.png">

This way, every single site could delete the data themselves if wanted.

Reply:
I disagree. As this is a developer only feature for, let's delete all data on sites. If you opt into it, then you should know what you are doing?

Reply:
_For now_ it is developer only. I could also see us removing this filter entirely in favor of the settings page.